Content.Server/Remotes/DoorRemoteSystem.cs

```diff
@@ -0,0 +1,92 @@
+using Robust.Shared.GameObjects;
+using Robust.Shared.Localization;
+using Robust.Shared.Player;
+using Robust.Shared.IoC;
+using Content.Shared.Interaction;
+using Content.Shared.Popups;
+using Content.Shared.Doors.Components;
+using Content.Shared.Doors.Systems;
+using Content.Shared.Physics;
+using Content.Shared.Access.Components;
+using Content.Server.Doors.Systems;
+using Content.Server.Doors.Components;
+
+namespace Content.Server.Remotes
+{
+    public sealed class DoorRemoteSystem : EntitySystem
+    {
+        [Dependency] private readonly SharedPopupSystem _popupSystem = default!;
+        [Dependency] private readonly SharedDoorSystem _sharedDoorSystem = default!;
+        [Dependency] private readonly DoorSystem _doorSystem = default!;
+        [Dependency] private readonly SharedInteractionSystem _interactionSystem = default!;
+
+        public override void Initialize()
+        {
+            base.Initialize();
+
+            SubscribeLocalEvent<DoorRemoteComponent, UseInHandEvent>(OnInHandActivation);
+            SubscribeLocalEvent<DoorRemoteComponent, AfterInteractEvent>(OnAfterInteract);
+        }
+
+        public void OnInHandActivation(EntityUid user, DoorRemoteComponent component, UseInHandEvent args)
+        {
+            switch (component.Mode)
+            {
+                case DoorRemoteComponent.OperatingMode.OpenClose:
+                    component.Mode = DoorRemoteComponent.OperatingMode.ToggleBolts;
+                    _popupSystem.PopupEntity(Loc.GetString("door-remote-switch-state-toggle-bolts"), args.User, Filter.Entities(args.User));
+                    break;
+                case DoorRemoteComponent.OperatingMode.ToggleBolts:
+                    component.Mode = DoorRemoteComponent.OperatingMode.OpenClose; // TODO: Swítch to ToggleEmergencyAcces when EA is implemented
+                    _popupSystem.PopupEntity(Loc.GetString("door-remote-switch-state-open-close"), args.User, Filter.Entities(args.User)); // TODO: See the above comment
+                    break;
+            /*
+                case DoorRemoteComponent.OperatingMode.ToggleEmergencyAccess:
+                    component.Mode = DoorRemoteComponent.OperatingMode.OpenClose;
+                    _popupSystem.PopupEntity(Loc.GetString("door-remote-switch-state-open-close"), args.User, Filter.Entities(args.User));
+                    break;
+            */
+            }
+        }
+        private void OnAfterInteract(EntityUid uid, DoorRemoteComponent component, AfterInteractEvent args)
+        {
+            if (args.Handled
+                || args.Target == null
+                || !EntityManager.TryGetComponent<DoorComponent>(args.Target, out var doorComponent) // If it isn't a door we don't use it
```

Comment:
Doesn't really matter, but just FIY: in systems there are some proxy methods that abbreviate these entity manager calls. It just makes it a bit nicer to read the code & results in shorter lines
e.g.:
`EntityManager.TryGetComponent` -> `TryComp`
`EntityManager.HasComponent` -> `HasComp`
`EntityManager.GetComponent` -> `Comp`